tests/driver.spec.js

```diff
@@ -37,7 +37,35 @@ describe('did:key method driver', () => {
       expect(kak.controller).to.equal(did);
       expect(kak.publicKeyBase58).to
         .equal('JhNWeSVLMYccCk7iopQW4guaSJTojqpMEELgSLhKwRr');
-    })
+    });
+
+    it('should resolve an individual key within the DID Doc', async () => {
+      const did = 'did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH';
+      const keyId = did + '#z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH';
+      const key = await didKeyDriver.get({did: keyId});
+
+      expect(key).to.eql({
+        '@context': 'https://w3id.org/security/v2',
+        id: 'did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH#z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH',
+        type: 'Ed25519VerificationKey2018',
+        controller: 'did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH',
+        publicKeyBase58: 'B12NYF8RrR3h41TDCTJojY59usg3mbtbjnFs7Eud1Y6u'
+      });
+    });
+
+    it('should resolve an individual key agreement key', async () => {
+      const did = 'did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH';
+      const kakKeyId = did + '#zBzoR5sqFgi6q3iFia8JPNfENCpi7RNSTKF7XNXX96SBY4';
+      const key = await didKeyDriver.get({did: kakKeyId});
+
+      expect(key).to.eql({
+        '@context': 'https://w3id.org/security/v2',
+        'id': 'did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH#zBzoR5sqFgi6q3iFia8JPNfENCpi7RNSTKF7XNXX96SBY4',
```

Comment:
Could omit the key quotes same as above.

Reply:
Thanks, removed.